ush/config_defaults.yaml

```diff
@@ -2275,9 +2236,9 @@ global:
   #
   DO_ENSEMBLE: false
   NUM_ENS_MEMBERS: 0
-  ENSMEM_NAMES: '{% for m in range(NUM_ENS_MEMBERS) %}{{ "mem%03d, " % m }}{% endfor %}'
-  FV3_NML_ENSMEM_FPS: '{% for mem in ENSMEM_NAMES %}{{ [EXPTDIR, "%s_%s" % FV3_NML_FN, mem]|path_join }}{% endfor %}'
-  ENS_TIME_LAG_HRS: '[ {% for m in range([1,NUM_ENS_MEMBERS]|max) %} 0, {% endfor %} ]'
+  ENSMEM_NAMES: '{% for m in range(global.NUM_ENS_MEMBERS) %}{{ "mem%03d, " % m }}{% endfor %}'
+  FV3_NML_ENSMEM_FPS: '{% for mem in global.ENSMEM_NAMES %}{{"%s/%s_%s" % workflow.EXPTDIR, workflow.FV3_NML_FN, mem }}{% endfor %}'
```

Comment:
I'm appreciating the explicit namespace information in these references. It's a little more text but IMO well worth it to make it clear where each of these values comes from.